cucumber/api/features/support/rest_helpers.rb

```diff
@@ -43,12 +43,35 @@ def patch_json path, body = nil, options = {}
     set_result result
   end
 
-  def get_json path, options = {}
+  def get_json(path, options = {})
     path = denormalize(path)
     result = rest_resource(options)[path].get
     set_result result
   end
 
+  # TODO: Add proper fix for this with real refactor of test code
```

Comment:
Do we have an issue filed for this?

Reply:
This one is better left as a TODO.  If we ever fix it, it will be as part of the general cleanup of these tests.  It's not worse than anything else that's here.  It was mostly a note to myself and other readers that this is not to be considered "ok".